identity-server/src/IdentityServer/Licensing/IdentityServerLicenseValidator.cs

```diff
@@ -92,9 +92,10 @@ private void EnsureAdded(ref HashSet<string> hashSet, object lockObject, string
 
     public void ValidateClient(string clientId) => ValidateClient(clientId, License);
 
-    HashSet<string> _clientIds = new();
-    object _clientIdLock = new();
-    bool _validateClientWarned = false;
+    private HashSet<string> _clientIds = new();
+    private object _clientIdLock = new();
+
+    private bool _validateClientWarned = false;
```

Comment:
Nitpick: it looks like the new line was added in the wrong place.

```suggestion
    private HashSet<string> _clientIds = new();
    private object _clientIdLock = new();
    private bool _validateClientWarned = false;

```